deta-lib/private/query.rkt

```diff
@@ -216,7 +216,7 @@
      (query schema opts (struct-copy ast:select stmt [offset (ast:offset n)]))]))
 
 (define order-by-pair/c
```

Comment:
How about `order-by-config`?

Reply:
eh, the tough art of naming things... that's an option, though I'm not sure it's the most crystal one because of the config files / command line flags association :/ I was also thinking along the lines of `column-order[ing]/c`, `column-sort/c`, etc., or things like `order-by-cell/c` (or `order-cell/c`, `ordering-cell/c`), `order{-by,ing}-condition/c`, `column-ordering/c`, `column-sort/c`, or even simply `ordering/c`. Or we may settle with `order-by-config/c` if you like it more.

Reply:
`ordering/c` sounds best to me out of all of those so let's go with that.